src/main/protobuf/soc/board.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package protos.soc;
+
+message Hex {
+    enum Resource {
```

Comment:
Do we want this to be distinct from the "inventory" resource enum?

Reply:
Well the Resource enum does not include Desert. I guess they could all be in one enum including MISC for port but I think having three separate enums is fine

Reply:
I think so too, but am going to do a little more thinking